src/scheduler/scheduler.py

```diff
@@ -2,18 +2,22 @@
 
 import logging
 import threading
+import json
 from contextlib import contextmanager
 from pathlib import Path
 
 from django.utils import timezone
 from requests_futures.sessions import FuturesSession
+from oauthlib.oauth2 import LegacyApplicationClient
```

Comment:
Are these imports still needed in this file: LegacyApplicationClient and OAuth2Session?